lib/LedgerSMB/Scripts/setup.pm

```diff
@@ -757,18 +756,27 @@ sub fix_tests{
 
     my $table = $request->{dbh}->quote_identifier($request->{table});
     my $edit = $request->{dbh}->quote_identifier($request->{edit});
-        my $where = $request->{id_where};
-    my $sth = $request->{dbh}->prepare(
-            "UPDATE $table SET $edit = ? where $where = ?"
+    my $where = $request->{id_where};
+    my $sthu = $request->{dbh}->prepare(
+        "UPDATE $table SET $edit = ? WHERE $where = ?"
+    );
+    my $sthi = $request->{dbh}->prepare(
+        "INSERT INTO table($edit,$where) VALUES (?, ?)"
     );
-
     for my $count (1 .. $request->{count}){
-        warn $count;
         my $id = $request->{"id_$count"};
-                $sth->execute($request->{"$request->{edit}_$id"}, $id) ||
-            $request->error($sth->errstr);
+        $sthu->execute($request->{"$request->{edit}_$id"}, $id) ||
+        $request->error($sthu->errstr);
+#        my $status = $sthu->execute($request->{"$request->{edit}_$id"}, $id);
+#        if ( $status == 0 ) {
+#            $sthi->execute($request->{"$request->{edit}_$id"}, $id) ||
+#            $request->error($sthi->errstr);
+#        } elsif ( $status < 0 ) {
+#            $request->error($sthu->errstr);
+#        }
```

Comment:
Can this commented out code be eliminated? Or is there a reason it should remain?
(Note: I'm not a fan of having commented out code)